packages/tasit-contracts/src/scripts/populateDecentralandContracts.js

```diff
@@ -78,263 +213,184 @@ const createEstate = async (estateContract, landContract, estate, wallet) => {
       const { args } = data;
       resolve(args._estateId);
     });
+
+    setTimeout(() => {
+      estateContract.unsubscribe();
+      reject();
+    }, EVENTS_TIMEOUT);
   });
 
   await action.waitForNonceToUpdate();
-  console.log("created id =", estateId);
+  console.log(`created id ${estateId}`);
 
   return estateId;
 };
 
-const createEstates = async (estateContract, landContract, estates, wallet) => {
+const createEstates = async estates => {
```

Comment:
So we're moving away from a purely functional style where all the args needed are passed in? I suppose that's fine for the purpose of this script, just noting it.

We could make a convenience function with just the one parameter call another function behind the scenes that grabs the other args from global constants so that each function remains more testable

Reply:
Yes, I moved out from functional style just because it's a self-contained script and for this case seemed to me that the code will be clearer if functions have few args.
I like your idea to wrap functional functions :+1: I'll keep that in mind for the future.

Reply:
Cool, sounds good! Resolving this